assets/elm/main.elm

```diff
@@ -912,6 +968,45 @@ nameTagView model =
         )
 
 
+votesView : Model -> Html Msg
+votesView model =
+    div []
+        [ h2 [] [ text "Who is the Dodgy Artist?" ]
+        , ul [] <| (playersExceptMeAndGameMaster model |> playerButtons)
+        ]
+
+
+playersExceptMeAndGameMaster : Model -> Dict.Dict String Player
+playersExceptMeAndGameMaster model =
+    Dict.remove model.playerId model.state.players |> removeGameMaster
+
+
+removeGameMaster : Dict.Dict String Player -> Dict.Dict String Player
+removeGameMaster players =
+    let
+        playerList =
+            players |> Dict.toList
+
+        gameMaster =
+            List.filter (\player -> (player |> Tuple.second |> .role) == GameMaster) playerList |> List.head
+    in
+    Dict.remove (gameMaster |> guaranteeTupleExists |> Tuple.first) players
+
+
+playerButtons : Dict.Dict String Player -> List (Html Msg)
+playerButtons players =
+    List.map
+        (\player ->
```

Comment:
pretty sure you can pattern match on the tuple directly here like
```elm
(\(playerId, playerRecord) ->
...

Reply:
Ah yes! I love it